source/js/components/people/people.scss

```diff
@@ -1,91 +0,0 @@
-#people,
-.fellow-list {
-  .headshot {
-    width: 100%;
-  }
-
-  .person-card {
-    border-top: 0.25rem solid black;
-  }
-
-  .person-issues {
-    line-height: 1;
-  }
-
-  .meta-block {
-    .meta-block-item {
-      display: flex;
-      align-items: center;
-      font-size: 12px;
-      font-weight: 900;
-
-      &::before {
-        width: 12px;
-        height: 12px;
-        display: block;
-        content: "";
-        margin-right: 5px;
-        background-size: contain;
-        background-repeat: no-repeat;
-      }
-    }
-
-    .meta-block-item-role {
-      &::before {
-        background-image: url("/_images/glyphs/mozilla-icon.svg");
```

Comment:
This image file should be removed as well as it's not being used anymore. I would search the file name (e.g., `mozilla-icon`) in the codebase to decide whether an image file can be removed or not.